### DISCUSSION TITLE
Why not use typescript ?

### DISCUSSION THREAD

**Discussion by YqxLzx** [nuejs/nue#36] [Q&A] [answered]
I think typescript will bring a lot of help to third-party libraries and their developers. For example, React.js and Vue.js both have good type support. Why doesn't Nuejs do this? Are there any other considerations?

**Comment by nobkd** [comment, marked as answer]
This looks like a duplicate of #10
Also see https://nuejs.org/faq/#ts

**Reply by YqxLzx** [reply]
THX YOU

**Comment by tipiirai** [comment]
Would @jsdoc type definitions solve the issue with third-party libraries and their developers?
